### ISSUE TITLE
Problem integrating AdMob adapter via Cocoapods

### ISSUE THREAD

**Issue by IgorRosocha** [mopub/mopub-ios-mediation#346]
1. **What is the issue?**
    I am unable to build project with `MoPub-AdMob-Adapters` pod.

2. **Check the applicable box(es) below by placing an `x` in the brackets:**
    - [X] I searched https://developers.mopub.com/ and found no answer.
    - [X] I searched https://twittercommunity.com/c/advertiser-api/mopub to make sure that this issue has not already been filed.
    - [X] I searched the adapter changelogs to make sure this issue has not already been resolved.

3. **Fill in the below template as detailed as possible:**
    * **Platform** (*Android, iOS, Unity*)
        iOS
    * **MoPub SDK Version**
        5.14.1
    * **Mediation SDK Name & Version**
        MoPub-AdMob-Adapters
    * **Mediation Adapter Version**
        7.66.0.1
    * **Steps to reproduce the issue** (include integration steps if not standard)
        Use pod `'MoPub-AdMob-Adapters', '7.66.0.1'`
    * **Observed behavior**
        After installing the pod and trying to compile the project, following error is received:

    ```swift
    Undefined symbols for architecture arm64:   
      "_OBJC_CLASS_$_GULUserDefaults", referenced from:   
          objc-class-ref in FirebaseInstallations(FIRInstallationsStore.o)  
          objc-class-ref in FirebaseInstanceID(FIRInstanceID.o)  
          objc-class-ref in FirebaseInstanceID(FIRInstanceIDCheckinPreferences.o)  
          objc-class-ref in FirebaseMessaging(FIRMessaging.o)  
          objc-class-ref in FirebaseMessaging(FIRMessagingPubSub.o)  
          objc-class-ref in FirebaseInstanceID(FIRInstanceIDUtilities.o)  
    ld: symbol(s) not found for architecture arm64  
    clang: error: linker command failed with exit code 1 (use -v to see invocation)
    ```
    From the log, it seems like error is related to Firebase. I have tried both `6.34.0` and `7.0.0`, both work correctly without `MoPub-AdMob-Adapters`, after adding the pod compilation fails. I have Firebase integrated using `Carthage`.
    
    * **Expected behavior** (if known)
        Project should be compilable with `MoPub-AdMob-Adapters` pod.

**Comment by chauduyphanvu** [comment]
@IgorRosocha, this seems to be the same issue as https://github.com/mopub/mopub-ios-mediation/issues/343. An adapter release should address it. Stay tuned!

Vu Chau
MoPub Team

**Comment by IgorRosocha** [comment]
@chauduyphanvu I have just tried the latest release `7.68.0.0`, and the issue still persists.

**Comment by chauduyphanvu** [comment]
@IgorRosocha, could you show us your Podfile and the results of installing the pods?

**Comment by IgorRosocha** [comment]
> @IgorRosocha, could you show us your Podfile and the results of installing the pods?

Podfile looks something like this, Firebase is integrated using Carthage:

```
platform :ios, '11.0'
project 'MyProject', 'AdHoc' => :release,'AppStore' => :release, 'Development' => :debug

inhibit_all_warnings!
use_frameworks!

def shared_pods
    pod 'SwiftLint', '~> 0.35'
    
    pod 'ACKLocalization', '~> 1.0'
    pod 'SwiftGen', '~> 6.0'
    pod 'Sourcery', '~> 0.16'
    
    pod 'Shimmer', '~> 1.0'
    pod 'gRPC-Swift', '1.0.0-alpha.20'
    pod 'BugfenderSDK', '~> 1.9'
end


target 'MyProject' do
    shared_pods
    pod 'MoPub-AdMob-Adapters', '~> 7.68.0.0'
    end
    
    target 'MyProjectTests' do
    shared_pods
    end
    
    target 'MyProjectWidget' do
    shared_pods
    end

    target 'MyProject_Core' do
        pod 'BugfenderSDK', '~> 1.9'
    end

    target 'Model' do
        pod 'gRPC-Swift', '1.0.0-alpha.20'
    end
end
```

Result of installing pod:
```
Analyzing dependencies
Downloading dependencies
Installing Google-Mobile-Ads-SDK (7.68.0)
Installing GoogleAppMeasurement (7.1.0)
Installing GoogleUserMessagingPlatform (1.3.0)
Installing GoogleUtilities (7.1.0)
Installing MoPub-AdMob-Adapters (7.68.0.0)
Installing PromisesObjC (1.2.11)
Installing mopub-ios-sdk (5.14.1)
Installing nanopb (2.30906.0)
Generating Pods project
Integrating client project
Pod installation complete! There are 8 dependencies from the Podfile and 35 total pods installed.
```

**Comment by chauduyphanvu** [comment]
@IgorRosocha, I did not see an issue when I added Firebase. This is my Podfile:

```
pod 'mopub-ios-sdk'
pod 'MoPub-AdMob-Adapters', '7.68.0.0'
pod 'Firebase/Analytics'	
pod 'Firebase/AppDistribution'	
pod 'Firebase/Auth'	
pod 'Firebase/Firestore'	
pod 'Firebase/Functions'	
pod 'Firebase/Messaging'	
pod 'Firebase/Storage'	
pod 'Firebase/Crashlytics'	
pod 'Firebase/DynamicLinks'	
pod 'Firebase/InAppMessaging'	
pod 'Firebase/MLVision'	
pod 'Firebase/MLModelInterpreter'	
pod 'Firebase/Performance'	
pod 'Firebase/Database'	
pod 'Firebase/RemoteConfig'	
```

This is my CocoaPods installation progress:

```
Analyzing dependencies
Downloading dependencies
Installing BoringSSL-GRPC (0.0.7)
Installing Firebase 7.1.0
Installing FirebaseABTesting (7.1.0)
Installing FirebaseAppDistribution (7.1.0-beta)
Installing FirebaseAuth (7.1.0)
Installing FirebaseCrashlytics (7.1.0)
Installing FirebaseDatabase (7.1.0)
Installing FirebaseDynamicLinks (7.1.0)
Installing FirebaseFirestore (7.1.0)
Installing FirebaseFunctions (7.1.0)
Installing FirebaseInAppMessaging (7.1.0-beta)
Installing FirebaseInstanceID (7.1.0)
Installing FirebaseMLCommon (7.1.0-beta)
Installing FirebaseMLModelInterpreter (7.1.0-beta)
Installing FirebaseMLVision (7.1.0-beta)
Installing FirebaseMessaging (7.1.0)
Installing FirebasePerformance (7.1.0)
Installing FirebaseRemoteConfig (7.1.0)
Installing FirebaseStorage (7.1.0)
Installing GTMSessionFetcher (1.5.0)
Installing Google-Mobile-Ads-SDK (7.68.0)
Installing GoogleAPIClientForREST (1.5.1)
Installing GoogleToolboxForMac (2.3.0)
Installing GoogleUserMessagingPlatform (1.3.0)
Installing GoogleUtilities 7.1.0
Installing MoPub-AdMob-Adapters (7.68.0.0)
Installing Protobuf (3.13.0)
Installing TensorFlowLiteC (2.3.0)
Installing TensorFlowLiteObjC (2.3.0)
Installing abseil (0.20200225.0)
Installing gRPC-C++ (1.28.2)
Installing gRPC-Core (1.28.2)
Installing leveldb-library (1.22)
Generating Pods project
Integrating client project
Pod installation complete! There are 19 dependencies from the Podfile and 43 total pods installed.
```

And this is the compiled app:

![image](https://user-images.githubusercontent.com/25616035/99280890-dc3c1180-27ff-11eb-9f90-349ce37250d7.png)

Are you able to validate without Carthage?

**Comment by IgorRosocha** [comment]
My bad - the issue has really been resolved by updating to `7.68.0.0`. 
I was accidentally trying it with `6.34.0` version of Firebase. After updating to `7.1.0`, it seems to work fine.

@chauduyphanvu thank you very much for your support, I am closing the issue now.
